Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -556,6 +672,36 @@ def project_list_command(client: Client, args: Dict) -> CommandResults:
         removeNull=True,
         headers=headers
     )
+
+    if partial_response:
+        partial_results = []
+        for result in results:
+            temp = {
+                'key': result.get('key'),
+                'uuid': result.get('uuid'),
+                'is_private': result.get('is_private'),
+                'name': result.get('name'),
+                'description': result.get('description'),
+                'created_on': result.get('created_on'),
+                'updated_on': result.get('updated_on'),
+                'owner': {
+                    'display_name': result.get('owner', {}).get('display_name'),
+                    'account_id': result.get('owner', {}).get('account_id')
+                },
+                'workspace': {
+                    'name': result.get('workspace', {}).get('name'),
+                    'slug': result.get('workspace', {}).get('slug')
+                }
+            }
+            partial_results.append(temp)
+
+        return CommandResults(
```

Comment:
No need to return 2 times the CommandResults - it's just a duplicate code.
Instead create a new var for example: `outputs = []`
and then 
if partial response: 
outputs = partial_results
else:
outputs = results
Then the commandResults will get `outputs=outputs`

Need to do this in all relevant places